### PR TITLE
Fix unit test for expiry emails on Monday

### PR DESCRIPTION
**Pull request by mdboom**

### CODE REVIEW

**Comment by fbertsch** [comment]
Ah, flake8 error.

File: tests/test_git_scraper.py [hunk 1]
```diff
@@ -280,7 +281,17 @@ def test_check_for_expired_metrics(expired_repo):
...
 
-    runner.main(cache_dir, out_dir, None, None, False, True, repositories_file, True, None, None)
+    # Mock `datetime.date.today` so it's a Monday, the only day that
+    # expirations are checked.
+    class MockDate(datetime.date):
+        @classmethod
+        def today(cls):
+            return datetime.date(2019, 10, 14)
+
+    with unittest.mock.patch("probe_scraper.glean_checks.datetime.date", new=MockDate):
```
Comment (fbertsch) [inline tests/test_git_scraper.py]: import `unittest.mock`, but use `unittest.mock.patch`? How is that working?

Comment (mdboom) [reply tests/test_git_scraper.py]: This is IIUC a standard way to import a python subpackage.  Could have alternatively done `from unittest import mock` -- happy to change if that's clearer.

Comment (fbertsch) [reply tests/test_git_scraper.py]: Oh right, that was my confusion. Carry on!

**Pull request merged**